core/modules/book/book.module

```diff
@@ -940,6 +948,8 @@ function book_node_insert(Node $node) {
     if ($destination['destination'] != 'admin/content/book/' . $node->book['bid'] && _book_outline_access($node)) {
       backdrop_set_message(t('Reorder book pages from the <a href="@url">book outline</a>.', array('@url' => url('admin/content/book/' . $node->book['bid']))));
     }
+
+    _book_cache_clear($node->book['bid'], TRUE);
```

Comment:
Same comment as in `book_node_update()`: 

This is not needed here, since the function `__book_cache_clear()` will be always invoked by `_book_update_outline()`, which is called above.

The result of adding this here is that the cache is cleared twice in a row for the same `bid`